### ISSUE TITLE
feat(ai-dev): PR_CREATED Handoff SurfaceでHuman Attentionを削減する (#1343 WS3)

### ISSUE THREAD

**Issue by s977043** [s977043/PlanGate#1346]
## Parent
#1343

## Dependency
WS0。WS2のprojection設計から再利用できる部分は再利用する。

## Goal
`ai-dev-workflow` のStable public contractを変えず、`PR_CREATED` 時のHuman handoff costを下げる。

既存終端:
```text
Request -> Plan -> C-1 -> C-2 -> Human C-3
        -> Execute -> Verify -> PR_CREATED
```

これは変更しない。

## Candidate surface
```text
PR_CREATED

Changed
Verified
Open
Risks / Unknowns
Human Attention Required
```

Human判断が必要な場合:
```text
Human Decision Required
- decision
- evidence
- options
- unresolved trade-off
```

## Principles
- execution logをhandoffへ転載しない
- self-report PASSだけを根拠にしない
- evidence refsを残す
- unresolved / unknownを隠さない
- handoff summaryはSSoTではない

## Rollout
presentation-only → dogfood → measure → contract changeの要否を再判断。

## Acceptance Criteria
- [ ] ai-dev public contract unchanged
- [ ] PR_CREATED unchanged
- [ ] Human C-3 / C-4 unchanged
- [ ] evidence traceability
- [ ] Human Attention Requiredが明示される
- [ ] no new approval semantics
- [ ] current outputsとのbefore/after評価

## Non-goals
ai-dev command chain変更、terminal state変更、auto merge、C-3自動化拡張。

**Comment by s977043** [comment]
## Phase 0 design refinement — workflow Skillization / review decision projection

Zenn article `9f8921926bb003` の実践（開発 workflow の Skill 化、Maker/Checker 分離、review decision の記録）を現行 PlanGate へ照合した。

### Adoption decision

**新しい `todays-work` 相当の monolithic Skill は追加しない。**

現行は既に以下へ責務分離されているため、記事の「人間が毎回 orchestration していた工程を AI workflow へ移す」という狙いは構造上ほぼ充足している。

- `workflow-conductor`: L-0〜V-4 の orchestration
- `ai-dev-plan`: plan
- `ai-dev-exec`: TDD execution
- `ai-dev-verify`: V-1〜V-4 + handoff
- `review-self.md` / `review-external.md`: reviewer findings
- `decision-log.jsonl`: append-only decision history
- `handoff.md`: WF-05 completion package

したがって WS3 では **workflow をもう1枚ラップするのではなく、既存 artifact から PR_CREATED Human Decision Surface を deterministic projection する**。

また、記事にある `review-decisions.md` 相当の新 artifact も初期導入では作らない。
既存の `review-external.md` + `decision-log.jsonl` と責務重複するため、まず projection で十分かを実証する。

---

## Source → Surface mapping

PR_CREATED の candidate surface:

```text
PR_CREATED

Changed
Verified
Open
Risks / Unknowns
Human Attention Required
Review Decisions
```

各 section は新しい truth を生成せず、既存 artifact を projection する。

| Surface | Primary source | Rule |
| --- | --- | --- |
| Changed | git diff / changed files + status/handoff | 実変更のみ。execution log を転載しない |
| Verified | V-1〜V-4 evidence / handoff requirementCompliance / testResults | self-report PASS のみは禁止。evidence ref 必須 |
| Open | known issues + unresolved review findings | unresolved / skipped / unavailable を隠さない |
| Risks / Unknowns | decision-log + known issues + verification uncertainty | 事実と推測を分離 |
| Human Attention Required | existing human-owned boundary / unresolved trade-off / policy-required human action | 新しい Judge / Gate を作らない |
| Review Decisions | review findings + decision-log disposition | finding → disposition → rationale → evidence ref を投影 |

### Review Decisions projection

新規 SSoT は作らず、概念的には以下を表示する。

```yaml
- finding_id: R-001
  reviewer: codex
  severity: minor
  disposition: accepted | rejected | deferred | unresolved
  rationale: ...
  evidence_ref: ...
```

初期実装では Markdown presentation-only。
schema / mandatory artifact 化は dogfood 後に必要性を再判断する。

---

## Implementation slices

### Slice 0 — Design freeze (this comment)

- source-of-truth mapping を固定
- no-new-Skill / no-new-review-decisions-artifact を固定
- projection-only / no semantic change を固定

### Slice 1 — Candidate renderer (River Review evidence 後)

最小変更候補:

1. `ai-dev-verify` に PR_CREATED Decision Surface の生成規約を追加
2. bundled `references/handoff.md` と上流 `docs/working/templates/handoff.md` に **optional candidate section** を追加
3. 現行 handoff 必須6要素は変更しない
4. `handoff-summary.schema.json` は変更しない
5. `PR_CREATED` / C-3 / C-4 / approval semantics は変更しない

### Slice 2 — Paired dogfood

同一 task の underlying evidence に対して:

- Baseline: current handoff
- Candidate: current handoff + Decision Surface

を比較する。

最低 guard:

- material visibility regression = 0
- critical/major visibility >= baseline
- unresolved / unknown visibility = 100%
- provenance loss = 0
- no new gate / state / approval regression

Human Attention は単独採用条件にしない。

### Slice 3 — Adopt / Revise / Reject

dogfood 後にのみ以下を判断:

- candidate section を正式化するか
- `handoff-summary.schema.json` へ structured projection を追加する価値があるか
- review disposition 専用 schema が本当に必要か

---

## Dependency update

River Review 側:

- #2369 ADR: accepted
- #2371 Decision Surface: accepted
- #2378 evaluation: in progress
- #2398 Phase A fixture/oracle SSoT: pending
- #2382 paired runner: pending

よって **WS3 の runtime/template implementation はまだ開始しない**。
ただし Phase 0 design freeze は先行可能であり、本コメントで完了扱いとする。

---

## Review of this plan

### Strong

1. **既存 architecture と整合**
   - article の workflow Skill 化をそのままコピーせず、既存 workflow-conductor / ai-dev skills を再利用する。
2. **SSoT 増殖を防ぐ**
   - `review-decisions.md` を新設せず、review finding と decision-log を source にする。
3. **Stable contract を維持**
   - PR_CREATED / C-3 / C-4 / handoff 6要素 / schema を初期段階で変更しない。
4. **Maker / Checker / Decision の責務分離を維持**
   - reviewer finding をそのまま decision に昇格させない。
5. **記事の価値は取り込む**
   - 人間が複数AI/工程を直接 orchestrate する負担を減らし、Human には decision surface を出す。

### Risks / corrections

- **Risk: handoff.md の肥大化**
  - Decision Surface は L1 projection とし、既存6要素の再掲を避ける。
- **Risk: reviewer finding と decision-log の join が曖昧**
  - 初期 dogfood では stable finding ID があるケースを優先。ID が無い場合に推測 join はしない。
- **Risk: Human Attention Required が新しい Judge になる**
  - trigger は既存 human-owned boundary / unresolved state の projection のみ。
- **Risk: article由来の「1 Skill化」を目的化する**
  - PlanGate では orchestration の一枚化ではなく、責務分離された Skill/Agent を conductor が束ねる現行形を維持する。

### Verdict

**Phase 0: GO. Runtime/template implementation: WAIT for #2378 evidence.**

この順序なら、記事の良い部分を取り込みつつ、PlanGate の既存責務境界と Human-owned authority を壊さない。

**Comment by s977043** [comment]
## Dependency evidence update — River Review Phase A deterministic PASS

River Review #2378 / #2382 が一段進んだ。

Frozen baseline/candidateを実runnerで10ケース paired実行し:

- deterministic safety: **PASS**
- candidate material miss: **0**
- structural regression: **0**
- raw baseline/candidate outputs: artifact保存済み

一方、candidateは additive v1 のため initial visible lines が最大 +5 行増えている。
したがって現時点のEvidenceは:

```text
Material Visibility Safety
        ✅ established

Human Decision Extraction
        ⏳ pending

Human Attention Reduction
        ⏳ unproven
```

### #1346 gate decision remains unchanged

**PR_CREATED runtime/template implementationはまだ開始しない。**

理由:
- #1346 の目的は単に情報を追加することではなく、Human Attentionを安全に圧縮すること
- River Review candidateはmaterial visibilityを改善したが、attention cost改善はまだ証明していない
- additive surfaceをそのままPlanGate handoffへ横展開すると、handoff肥大化をコピーする可能性がある

次の入力Evidenceは #2378 Human Decision Extraction scoring / Phase B判断。
それまでは Phase 0 design freeze のまま維持する。
